pages/activity/[id].js

```diff
@@ -3,17 +3,19 @@ import styled from "styled-components";
 import Link from "next/link";
 import { data } from "@/lib/db/db";
 import { useRouter } from "next/router";
+import { StyledLink } from "@/components/StyledLink";
 
-export default function SingleActivity({ editTag }) {
+export default function SingleActivity({ editTag, removeActivity }) {
   const router = useRouter();
   const id = router.query.id;
   const activity = data.find((activity) => activity.id == id);
+
   if (!activity) {
     return <div>not found.....</div>;
   }
   return (
     <>
-      <ul>
+      <ul className="form">
```

Comment:
Another CSS class that should be a styled component. And the name form does not make sense if you do not style a form here. Please choose another name. :)